rt/frontend/jaxrs/src/main/java/org/apache/cxf/jaxrs/utils/InjectionUtils.java

```diff
@@ -1526,4 +1529,13 @@ public static Type processGenericTypeIfNeeded(Class<?> serviceCls, Class<?> para
     public static Object getEntity(Object o) {
         return o instanceof GenericEntity ? ((GenericEntity<?>)o).getEntity() : o;
     }
+
+    public static Set<Class<?>> getCustomContextClasses() {
```

Comment:
Is this utility OK to include here?  Or should I move it into the CDI module?

Reply:
Moving to CDI module would be better I believe, so it would be clear where this flow belongs to

Reply:
Sure, but just to confirm the actual interface `ContextClassProvider` stays in frontend/jax-rs since multiple modules will use it (and shouldn't rely on CDI)